src/components/badges/Badge.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Text, {SIZE as TEXT_SIZE, WEIGHT as TEXT_WEIGHT} from '../text/Text';
+
+const SIZE = {
+  NORMAL: 'normal',
+  SMALL: 'small',
+  LARGE: 'large'
+};
+
+const COLOR = {
+  NORMAL: {BADGE: 'light', TEXT: 'default'},
```

Comment:
text colors are defined in Text component, you should not use strings here but the constants. Do sth like this instead:

```javascript
import Text, {COLOR as TEXT_COLOR} from '../text/Text';
//...
NORMAL: {BADGE: 'light', TEXT: TEXT_COLOR.DEFAULT},
```